lib/drivers/node-mongodb-native/connection.js

```diff
@@ -84,9 +87,9 @@ NativeConnection.prototype.useDb = function(name) {
 
   newConn.name = name;
 
-  // push onto the otherDbs stack, this is used when state changes
-  this.otherDbs.push(newConn);
-  newConn.otherDbs.push(this);
```

Comment:
@vkarpov15 Do you want to keep the original behaviour of the ```otherDbs``` prop

Reply:
Yes please, lets do that